_posts/2019-11-12-docker-rename-to-container.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Pulp Docker plugin will be renamed to Pulp Container plugin in Pulp3
+author: Ina Panova
+tags:
+  - docker
+  - container
+---
+
+With Pulp 3 GA pulp_container 1.0 will come along.
+
+## Why renaming pulp_docker to pulp_container in Pulp3?
+
+* pulp_docker plugin implies management of docker format images only. This plugin will stay in Pulp 2.
```

Comment:
s/will stay in/will remain/

Reply:
I think Docker should be capitalized everywhere. At the very least, capitalization should be consistent throughout.